src/sssom/context.py

```diff
@@ -52,10 +52,13 @@ def _get_built_in_prefix_map() -> Converter:
 HINT = Union[None, PrefixMap, Converter]
 
 
-def ensure_converter(prefix_map: HINT = None) -> Converter:
+def ensure_converter(prefix_map: HINT = None, *, use_bioregistry: bool = True) -> Converter:
     """Ensure a converter is available."""
     if not prefix_map:
-        return get_converter()
+        if use_bioregistry:
+            return get_converter()
```

Comment:
shouldnt this be a chain of built-in prefix map with get_converter()? I cant see a use case in which we would ever not want the built in prefixes to be added first.

Reply:
`get_converter` has the built-in prefix map chained inside it as expected

Reply:
Sweet.

Reply:
Addressed in 20fb6d5. I also wrote more detailed documentation for this function.